src/clients/inst_autosetup.rb

```diff
@@ -188,6 +188,12 @@ def main
           ["Import", Ops.get_map(Profile.current, "add-on", {})]
         )
         Call.Function("add-on_auto", ["Write"])
+        # The entry "kexec_reboot" in the Product description can be set
+        # by the AutoYaST configuration setting (general/forceboot) and should
+        # not be reset by any other Product description file.
+        # So we set it here again.
```

Comment:
Oh, sorry, I see that the patch `AutoinstGeneral.write` does not touch that setting if it's not specified.